### DISCUSSION TITLE
[A-L] (2023/24) Foglio 11 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#334] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-23 104005](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3b40e065-4224-4757-88a9-323f90e71c70)

**Comment by Elia-Belli** [comment]
- I vettori $(\underline v_1, \underline v_2,\underline v_3)$ sono linearmente indipendenti se e solo se il seguente sistema ammette unica soluzione nulla:

$$\begin{cases}x=0\\
y=0\\
x+z=0\end{cases}\iff\begin{cases}x=0\\
y=0\\
z=0\end{cases}$$

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a685041f-e098-43fe-8b49-33287b4c49ae)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/09fa3540-5f0c-4e83-a4a9-5be888f3f3c9)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/de674dea-4a67-4620-a932-bbb3d7fafb05)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/72560b05-9044-4380-9a9d-a0837013d1f9)

**Comment by 18xChris** [comment]
> ![image](https://private-user-images.githubusercontent.com/79638284/286864866-72560b05-9044-4380-9a9d-a0837013d1f9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE3Njc5OTksIm5iZiI6MTcwMTc2NzY5OSwicGF0aCI6Ii83OTYzODI4NC8yODY4NjQ4NjYtNzI1NjBiMDUtOTA0NC00MzgwLTlhOWQtYTA4MzcwMTNkMWY5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjA1VDA5MTQ1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZiNjMzMmYwNzJiZDM3NjVjNTJmZTM2OTk5ZTcxYWZiMjAxYzdjYzQxMTYyZmVlZjIyMmJjNTM5MGY3NDU2ZmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.RajMczgq6Ux_3jQ1Ot6ov49dlmPVlWWqx0MrUxBL9Tg)

Non riesco a capire perché hai eseguito quell'operazione nell'ultima frase che hai scritto (quella rossa
)

**Comment by CiottoloMaggico** [comment]
È solo per dire che v1 e v2 bastano per generare tutto W. Infatti fissando i coefficienti puoi riscriverti i vettori dati in "input" allo span.
